src/labels/CustomLabels.labels

```diff
@@ -7583,6 +7591,94 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Gift Processing configuration error</shortDescription>
         <value>Your org is missing required configuration for Gift Processing. Contact your admin for help.</value>
     </labels>
+    <labels>
+        <fullName>gsApplicationStatus</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Application Status label for the application status component</shortDescription>
+        <value>Application Status: </value>
+    </labels>
+    <labels>
+        <fullName>gsApplicationStatusModalHeader</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Header of the modal label for the application status component</shortDescription>
+        <value>Check Application Status</value>
+    </labels>
+    <labels>
+        <fullName>gsApplyForFreeLicenses</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Apply for licencses button label for the application status component</shortDescription>
+        <value>Apply for Salesforce Licenses</value>
+    </labels>
+    <labels>
+        <fullName>gsCheckStatus</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Check Status label for the application status component</shortDescription>
+        <value>Check Status</value>
+    </labels>
+    <labels>
+        <fullName>gsClose</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Close label in the modal for the application status component</shortDescription>
+        <value>Close</value>
+    </labels>
+    <labels>
+        <fullName>gsDaysAdded</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Days added label for the application status component</shortDescription>
+        <value>30 Days Added</value>
+    </labels>
+    <labels>
+        <fullName>gsDaysRemainingInFreeTrial</fullName>
+        <categories>Getting Started Page</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>gsDaysRemainingInFreeTrial</shortDescription>
```

Comment:
I'd recommend using a short desc that is similar to the others: Days remaining in the free trial label for the application status component